server/embed/config_test.go

```diff
@@ -136,14 +139,25 @@ func TestConfigFileFeatureGates(t *testing.T) {
 			expectedFeatures: map[featuregate.Feature]bool{
 				features.StopGRPCServiceOnDefrag: false,
 				features.DistributedTracing:      false,
+				features.InitialCorruptCheck:     false,
 			},
 		},
 		{
-			name:                   "can set feature gate to true from feature gate flag",
+			name:                   "can set feature gate StopGRPCServiceOnDefrag to true from feature gate flag",
 			serverFeatureGatesJSON: "StopGRPCServiceOnDefrag=true",
 			expectedFeatures: map[featuregate.Feature]bool{
 				features.StopGRPCServiceOnDefrag: true,
 				features.DistributedTracing:      false,
+				features.InitialCorruptCheck:     false,
+			},
+		},
+		{
+			name:                   "can set feature gate InitialCorruptCheck to true from feature gate flag",
+			serverFeatureGatesJSON: "InitialCorruptCheck=true",
+			expectedFeatures: map[featuregate.Feature]bool{
+				features.StopGRPCServiceOnDefrag: false,
+				features.DistributedTracing:      false,
+				features.InitialCorruptCheck:     true,
```

Comment:
Can you add a similar test case for setting `--experimental-initial-corrupt-check` command line flag resulting in feature gate being flipped?

Reply:
Updated.

I can see that we have a test for `ExperimentalFlags to FeatureGates`, this is the reason i have not add this test case here.  `TestSetFeatureGatesFromExperimentalFlags`

https://github.com/etcd-io/etcd/blob/cb0cd4b21882c922b843b0b7e1ad03e9630a2f01/server/embed/config_test.go#L782

Do we still need to add corresponding conversion tests here for each ExperimentalFlags?